packages/client/screens/MySettings/settingsMachine.ts

```diff
@@ -323,6 +342,114 @@ export const settingsMachine =
+                                                actions: forwardTo(
+                                                    'retrieveUserGoogleAccesTokenMachine',
+                                                ),
+                                            },
+
+                                            __RETRIEVED_USER_GOOGLE_ACCESS_TOKEN_SUCCESSFULLY:
+                                                {
+                                                    actions:
+                                                        settingsModel.assign({
+                                                            userGoogleAccessToken:
+                                                                (
+                                                                    _context,
+                                                                    {
+                                                                        userGoogleAccessToken,
+                                                                    },
+                                                                ) =>
+                                                                    userGoogleAccessToken,
+                                                        }),
+                                                    target: 'Persist user google access token to server',
+                                                },
+
+                                            __FAILED_TO_RETRIEVE_USER_GOOGLE_ACCESS_TOKEN:
+                                                {
+                                                    target: '.Retrieve user google access token failed',
+                                                },
+                                        },
+                                    },
+
+                                'Persist user google access token to server': {
+                                    invoke: {
+                                        src: 'sendUserLinkGoogleAccount',
+
+                                        onDone: [
+                                            {
+                                                cond: 'linkGoogleAccountErrorRetrievedGoogleAccountUnavailable',
+
+                                                actions:
+                                                    'Display toast error link google account server error google id unavailable',
+
+                                                target: "#Settings.Fetched user's settings.Link Google account manager.Waiting for user to start a link google account operation",
+                                            },
+                                            {
+                                                actions: [
+                                                    'Display link google account operation success toast',
+                                                    'reset user google access token from context',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I think there is a typo. The action should be _Reset user google access token from context_. (The first _r_ is capitalized.)

Reply:
Ah nice one, thx about to be updated !